src/App.jsx

```diff
@@ -1,44 +1,19 @@
 import React from 'react';
 import 'bulma/css/bulma.css';
+import { Article } from './components/Article';
 
 export const firstArticle = {
   title: 'Electric-propulsion startup Orbion raises $9.2 million Series A',
-  date: new Date('2019-08-30'),
+  date: new Date('2022-09-08'),
```

Comment:
There is no need to change the date. Tests don't like it)
![image](https://user-images.githubusercontent.com/96409563/228475667-e381c3ae-9774-4896-9c72-9f7ec4ad4f72.png)